facdb/cli.py

```diff
@@ -44,13 +44,19 @@ def build():
     """
     Building facdb based on facdb_base
     """
+    print("create facdb geometry")
```

Comment:
don't think these print statements should be merged

Reply:
I thought about not adding it. But I think it is kind of helpful for debugging when I was trying to locate where the records got dropped for the new version nysdec_lands

Reply:
my approach for print statements for debugging is keep them on my local machine and not merge. We could merge them into our build branch too, if you merge yours I will merge mine as well. @mbh329 what do you think?